src/attack_flow_builder/src/store/Stores/HotkeyStore.ts

```diff
@@ -134,6 +134,21 @@ export default {
                     shortcut: edit.duplicate,
                     repeatable: false
                 },
+                {
+                    data: () => new App.ShowFindDialog(ctx),
+                    shortcut: edit.find,
+                    repeatable: false
+                },
+                // {
```

Comment:
You have shortcuts assigned to these actions but they're still commented out. Is this intended?

Reply:
Whoops -- meant to enable those. I ended up going with F3 for "Find Next" and Shift+F3 for "Find Previous" to match how notepad.exe works. (On MacOS it would usually be Cmd+G and Cmd+Shift+G, I think, but that might conflict with the "Grid" hotkey.)